modules/33acrossBidAdapter.js

```diff
@@ -98,12 +106,111 @@ function _createSync(siteId) {
   }
 }
 
-function _getFormatSize(sizeArr) {
+function _getSize(size) {
   return {
-    w: sizeArr[0],
-    h: sizeArr[1],
-    ext: {}
+    w: parseInt(size[0], 10),
+    h: parseInt(size[1], 10)
+  }
+}
+
+function _getMinSize(sizes) {
+  return sizes.reduce((min, size) => size.h * size.w < min.h * min.w ? size : min);
+}
+
+function _getBoundingBox(element, { w, h } = {}) {
+  let { width, height, left, top, right, bottom } = element.getBoundingClientRect();
+
+  if ((width === 0 || height === 0) && w && h) {
+    width = w;
+    height = h;
+    right = left + w;
+    bottom = top + h;
+  }
+
+  return { width, height, left, top, right, bottom };
+}
+
```

Comment:
Can you elaborate what `_transformSizes` does? Assuming that the `sizes` from the `bidRequest` object will something like following:
```
[[320, 50], [320,100]...]
```

Reply:
Also how do you think we should treat an undefined size? Don't recall how Prebid core treats sizes not being set in the ad unit parameters. If it simply throws errors, should we then include checking for size in `isBidRequestValid`

Reply:
Because GPT supports 2 size formats:
1. [w, h]
2. [[w, h], [w, h]]

https://support.google.com/admanager/answer/1638622